pkg/GAPJulia/JuliaInterface/julia/gap_to_julia.jl

```diff
@@ -9,11 +9,23 @@ The parameter `recursion_dict` is meant to preseve egality
 of converted objects and should never be given by the user.
 """
 
+## Default for conversion:
+## Do not convert Julia types, as long as type matches
+## If you have a GAP list of Julia Tuples, this function
+## enables conversion of that list to a list of type
+## Array{Tuple,1}.
+function gap_to_julia(t::T, x::Any) where T <: Type
+    if ! (typeof(x) <: t)
+        throw(MethodError("Wrong type: expected " * string(t) * ","))
+    end
+    return x
+end
+
 ## Default
 gap_to_julia(::Type{GAPInputType},x::GAPInputType) = x
```

Comment:
Is this method still needed now? It is a special case of the above, isn't it? (Might still be needed due to its slightly higher method priority, I guess?)

Reply:
Indeed, that one is not necessary anymore :).